crowdin/model/workflows.go

```diff
@@ -75,6 +75,75 @@ type WorkflowTemplatesListResponse struct {
 	Data []*WorkflowTemplateResponse `json:"data"`
 }
 
+// WorkflowStepString represents a string on a workflow step.
+type WorkflowStepString struct {
```

Comment:
This is the same string as the [`SourceString`](https://github.com/crowdin/crowdin-api-client-go/blob/main/crowdin/model/source_strings.go#L10)